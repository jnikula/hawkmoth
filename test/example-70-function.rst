
.. c:function:: int frob(struct list *list, enum mode mode)

   List frobnicator.

   :param list: The list to frob.
   :param mode: The frobnication mode.
   :return: 0 on success, non-zero error code on error.
   :since: v0.1


.. c:function:: int frobo(const char *fmt, ...)

   variadic frobnicator

   :param fmt: the format
   :param ...: variadic

