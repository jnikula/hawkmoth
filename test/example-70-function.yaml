directive-arguments:
- example-70-function.c
example-title: Function
