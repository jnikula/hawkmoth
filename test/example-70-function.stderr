WARNING: example-70-function.c:9: declaration of 'struct list' will not be visible outside of this function
WARNING: example-70-function.c:9: declaration of 'enum mode' will not be visible outside of this function
